gt/eval/ONB_ibn_19110701_012/ONB_ibn_19110701_012.tif_tl_2.gt.txt

```diff
@@ -1 +1 @@
-Innsbrucker Nachrichten
\ No newline at end of file
+Jnnsbrucker Nachrichten
```

Comment:
As "I" and "J" look identical in Fraktur, "Innsbrucker" might be the better transcription. Ideally the neural network could learn when to recognize "J" or "I".

Reply:
Changed J -> I in Fraktur now.